src/main/resources/resources/logisim/strings/std/std_de.properties

```diff
@@ -774,6 +774,7 @@ TTL7414 = 7414: Sechsfach Invertierer (Schmitt-Trigger)
 TTL74161 = 74161: synchroner 4-bit-Zähler mit Clear
 TTL74163 = 74163: synchroner 4-bit-Zähler mit synchronem Clear
 TTL74165 = 74165: 8-Bit Parallel zu Serielles Schieberegister
```

Comment:
"Parallel zu Serielles" -> "parallel-zu-seriell"

Reply:
will fix in upcoming PR to fix #1472